src/prebid.js

```diff
@@ -655,4 +698,16 @@ pbjs.enableSendAllBids = function () {
   pb_sendAllBids = true;
 };
 
+pbjs.disableSendAllBids = function () {
```

Comment:
The spec for `enableSendAllBids` was as a "one way" config as it seemed there was no case where turning this off was needed once it was turned on. Are you aware of such of a case?